Modify the Service Composition guide for new Ballerina version (#380)
wso2/ballerina-integrator: docs/content/src/learn/guides/testing/writing-tests-using-data-providers/README.md

The backend has been implemented to accept user input and provide an output message. 

## Developement

> It is implemented a service as 'hello' which has the function as sayHello. As an example, if user provides an input as 'John', it will send the response to the client as 'Hello John'.

Reviewer: ```suggestion
A service called `hello` is implemented, which has a function called `sayHello`. For example, if you provide an input 'John', it will send the response to the client as 'Hello John'.
```
